vnext/Microsoft.ReactNative/Utils/TransformableText.h

```diff
@@ -7,13 +7,13 @@ namespace Microsoft::ReactNative {
 struct TransformableText final {
   enum class TextTransform : uint8_t { Undefined, None, Uppercase, Lowercase, Capitalize };
 
-  TextTransform textTransform{TextTransform::Undefined};
-  std::wstring originalText{};
-
-  std::wstring TransformText() const noexcept {
+  static winrt::hstring TransformText(
```

Comment:
Was trying to minimize the diff here, but if we want to, we can lift this out of the TransformableText struct (and eliminate the TransformableText struct).